### DISCUSSION TITLE
requesting for the review.

### DISCUSSION THREAD

**Discussion by pbasavaraju** [ipxe/ipxe#444] [Q&A]
Dear iPXE developers,

After creating the PULL #370 didn't heard anything like review comments or time line to merge the pull request with main branch.
our customers are waiting to use this solution and asking for time line to have this solution in iPXE main branch. please assist on the process to be followed to get it through.  

thanks & regards,
Pavan B

**Comment by berlin4apk** [comment]
Maybe it will be easier to review it if @mcb30 getting the hardware form Marvell?
(I'm not knowing if it was so a offer or request already done)

**Reply by pbasavaraju** [reply]
we didn't received any replay after creating #370 pull request. this pull request is created after adding fixing all review comments we received for pull request #328. we didn't got any request for hardware as well.

**Reply by berlin4apk** [reply]
Reading the comments in #370
This question should be closed.
